File: src/IsbInterop/Tasks/IAttachmentList.vb
Imports IsbInterop.Accessory
Imports IsbInterop.Base

Namespace Tasks
  ''' <summary>
  ''' Список вложений.
  ''' </summary>
  Public Interface IAttachmentList
    Inherits IForEach(Of IAttachment)
    ''' <summary>
    ''' Добавить вложение.
    ''' </summary>
    ''' <param name="attachmentInfo">Информация о добавляемом вложении.</param>
    Sub Add(attachmentInfo As IObjectInfo)
  End Interface
End NameSpace

File: src/IsbInterop/Tasks/IAttachmentList.cs
using IsbInterop.Accessory;
using IsbInterop.Base;
using IsbInterop.Collections;

namespace IsbInterop.Tasks
{
  /// <summary>
  /// Список вложений.
  /// </summary>
  public interface IAttachmentList : IForEach<IAttachment>
  {
    /// <summary>
    /// Добавить вложение.
    /// </summary>
    /// <param name="attachmentInfo">Информация о добавляемом вложении.</param>
    void Add(IObjectInfo attachmentInfo);
  }
}
